ipatests/test_integration/test_user_permissions.py

```diff
@@ -190,6 +190,37 @@ def test_user_add_withradius(self):
              '--user-auth-type', 'radius', '--radius-username', 'testradius',
              'testradius', '--radius', radiusproxy])
 
+    def test_user_add_fix_capitalization_error(self):
+        """
+        Test that an existing user which has been modified using ipa user-mod
+        and has the same first and last name beginning with caps does not
+        throw the error 'ipa: ERROR: Type or value exists:' and i
+        instead gets modified
+
+        This is a test case for Pagure issue
+        https://pagure.io/freeipa/issue/5879
+        """
+        # kinit admin
+        tasks.kinit_admin(self.master)
+
+        # Create an ipa-user
+        ipauser = 'ipauser1'
+        first = 'ipauser'
+        modfirst = 'Ipauser'
+        last = 'test'
+        modlast = 'Test'
+        password = 'Secret123'
+        password_confirmation = "%s\n%s\n" % (password, password)
+        self.master.run_command(
+            ['ipa', 'user-add', ipauser, '--first', first, '--last', last,
+             '--password'],
+            stdin_text=password_confirmation)
```

Comment:
I would write stdin_text like this:
stdin_text= "%s\n%s\n" % (password, password)
or
stdin_text= "Secret123\nSecret123\n" as password is not used anywhere else.